ci/licenses_golden/tool_signature

```diff
@@ -1,2 +1,2 @@
-Signature: 2670e7ab781a8bc009ac069d3b7d9cb5
```

Comment:
non blocking where does this hash come from?

Reply:
No idea, CI tells me about it